.github/workflows/build.yaml

```diff
@@ -0,0 +1,32 @@
+name: Build
+
+on:
+  pull_request:
+  push:
+
+defaults:
+  run:
+    shell: 'bash -Eeuo pipefail -x {0}'
```

Comment:
```suggestion
```
There are no jobs/steps using shell in this workflow.

Reply:
Fixed! Thanks!